Documentation/guides.xml

```diff
@@ -9,6 +9,11 @@
         class="\T3Docs\Typo3DocsTheme\DependencyInjection\Typo3DocsThemeExtension"
         edit-on-github="TYPO3-Documentation/render-guides"
         edit-on-github-branch="main"
+        typo3-core-preferred="stable"
     />
-    <inventory id="h2document" url="https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/" />
+    <!-- explicitly link to version 8.7 of TYPO3 Explained -->
```

Comment:
Maybe add a comment, that they are really used in the chapter "Interlink inventories", so it is clear, what happens here.

Reply:
done